crates/amaru-stores/src/rocksdb/consensus/mod.rs

```diff
@@ -370,11 +370,16 @@ pub mod test {
     use amaru_ouroboros_traits::tests::{
         any_header_with_parent, any_headers_chain, make_header, run,
     };
+    use rocksdb::Direction;
     use std::collections::BTreeMap;
     use std::path::Path;
     use std::sync::Arc;
     use std::{fs, io};
 
+    /// "chain"
+    /// TODO: move to util once we are ready to implement chain tracking
+    const CHAIN_PREFIX: [u8; CONSENSUS_PREFIX_LEN] = *b"chain";
```

Comment:
oh nice, I didn’t that that this works! OTOH you only use `&CHAIN_PREFIX[..]`, so typing as `&[u8]` would be more direct (you can add an assertion about the length in the initializer, if I’m not mistaken)